index.html

```diff
@@ -3,8 +3,41 @@
 <head>
     <meta charset="UTF-8">
     <title>Tech It Easy Dashboard</title>
+    <link rel="stylesheet" href="styles.css">
 </head>
+
 <body>
+<div class="container">
+    <div class="header">
+        <h1>Tech-it-Easy | Financial Dashboard</h1>
+    </div>
+    <div class="main-numbers">
+        <h2>
+            <p>VOORRAAD: <span id="stock"> getal = totaal aantal tv's nog op voorraad</span> producten</p>
+            <p>DOELOMZET: <span id="revenue-goal"> bedrag = opbrengst bij verkoop alle voorraad</span></p>
+            <p>OMZET: <span id="revenue-now"> bedrag = opbrengst van verkoop tot nu toe</span></p>
+        </h2>
+    </div>
+    <div class="tv-types">
+        <h3>TWEE VOORBEELDEN VAN TV-TYPES:</h3>
+        <p id="tv1"> = willekeurige tv 1</p>
+        <p id="tv2"> = willekeurige tv 2</p>
+    </div>
+    <div class="all-tvs">
+        <div class="buttons">
+            <button class="btn" id="btn-0" onclick="getCompleteTvInfosString(inventory)">Show All</button>
+            <button class="btn" id="btn-1" onclick="getObjectsSortedbyPrice(inventory)">Sort Prices</button>
```

Comment:
Ja die knop miste eigenlijk. Maar eigenlijk zou je ze dan ook ongesorteerd moeten laten zien natuurlijk. Heb ik trouwens ook laten zitten, al had ik wel een copie van de lijst gemaakt in de sort om dit mogelijk te maken.